assets/css/focus.css

```diff
@@ -0,0 +1,41 @@
+/* For backwards compatibility, style :focus state so that it works in all browsers,
+but unstyle not-focus-visible focus state in browsers that support focus-visible */
+
+/* extra button selectors necessary to overwrite normalize.css */
```

Comment:
I assume leaving a bunch `/* */` of comments meant for current and future contributors is fine because the final output (`formatters/html/dist/*.css`?) doesn't include them. I usually work with SCSS where `//` comments get stripped but `/* */` comments stay in the output.